daemon/src/maker.rs

```diff
@@ -276,8 +276,10 @@ async fn main() -> Result<()> {
                     cfds,
                 )));
 
-                // use `.send` here to ensure we only continue once the update was processed
-                oracle_actor_address.send(oracle::Sync).await.unwrap();
+                oracle_actor_address
```

Comment:
:question: Should we get ahead of the likely problem caused by this and create an issue? I would expect it to cause an error when trying to create an order before the attestations are loaded.

Reply:
Announcements are still loaded on startup. A user would have to be really quick for them to not be available and even then, they can still retry and it will likely succeed.

Reply:
Yeah, it likely won't be a problem for users. I would expect manual testing to be influenced by this, but if we are happy to just retry then it's okay.